internal/services/signalr/web_pubsub_resource.go

```diff
@@ -155,6 +155,11 @@ func resourceWebPubSub() *pluginsdk.Resource {
 				Computed: true,
 			},
 
+			"ip_address": {
```

Comment:
This will need to be set in the read also?

Reply:
`ip_address` is removed as it's named as `external_ip`